tests/core/objc/BUILD.bazel.in

```diff
@@ -0,0 +1,20 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
A few changes needed here:

* Move to `tests/core/cgo/objc` or simply `tests/core/cgo`. Rename to BUILD.bazel.
* Check that `objc_copts` and `cppopts` gets passed in. It's unclear to me whether `copts` and `cxxopts` should be passed in. Probably no for now.
* Check that an `objc_library` can be in `cdeps`. It's fine to break this into multiple tests in the same directory by the way.
* Use a `select` expression or build constraints to make this build on all platforms. It will just be an empty test on Linux and Windows. Something like below.

```
go_test(
    name = "objc_test",
    srcs = ["adder_darwin_test.go"],
    embed = select({
        "@io_bazel_rules_go//go/platform:darwin": [":objc"],
        "//conditions:default": [],
    }),
)

go_library(
    name = "objc",
    srcs = [
        "add_darwin.h",
        "add_darwin.m",
        "adder_darwin.go",
        "adder_darwin_test.go",
    ],
    cgo = True,
    objc = True,
    objc_enable_modules = True,
)
```

Reply:
The main isssue is that simply referencing an `objc_library` in Linux seems to crash since there are no objc actions defined in the `CROSSTOOL`....
I don't think doing what you suggested will fix it ?

Reply:
The only way I see this working is via a workspace rule maybe ?